docs/examples/MarrmotM14.ipynb

```diff
@@ -2,204 +2,157 @@
  "cells": [
```

Comment:
<p>Image does not work before PR is merged. If I replace <code>main</code> with <code>update_marrmot_notebooks</code>   it still does not work. If I change it to <code>https://github.com/eWaterCycle/ewatercycle/raw/update_marrmot_notebooks/docs/examples/logo.png</code>  then it does work. Can URL be changed to <code>https://github.com/eWaterCycle/ewatercycle/raw/main/docs/examples/logo.png</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
thanks, done :+1: